client/app/scripts/utils/zoom-utils.js

```diff
@@ -1,5 +1,5 @@
 
 // See https://github.com/d3/d3-zoom/blob/807f02c7a5fe496fbd08cc3417b62905a8ce95fa/src/zoom.js
 export function defaultWheelDelta(ev) {
-  return ev.deltaY * (ev.deltaMode ? 120 : 1) * 0.002;
+  return ev.deltaY * (ev.deltaMode ? 15 : 1) * 0.0035;
```

Comment:
Maybe add a comment to elaborate that we are checking `deltaMode` to see whether `deltaY` is in pixel or lines. It seems like only FF uses `1` (`DOM_DELTA_LINE`) while other browsers use `0` (`DOM_DELTA_PIXEL`).